plugins/scaffolder/src/components/ListTasksPage/ListTasksPage.tsx

```diff
@@ -44,6 +44,73 @@ import { useNavigate } from 'react-router-dom';
 import { useTranslationRef } from '@backstage/core-plugin-api/alpha';
 import { scaffolderTranslationRef } from '../../translation';
 
+interface ListTasksTableProps {
+  page: number;
+  setPage: Dispatch<SetStateAction<number>>;
+  limit: number;
+  setLimit: Dispatch<SetStateAction<number>>;
+  tasks?: ScaffolderTask[];
+  totalTasks?: number;
+}
+
+export function ListTasksTable({
```

Comment:
Think we wanna move this `ListTasksTable` to `scaffolder-react` plugin instead and export it from there and use it here.